android/build.gradle

```diff
@@ -22,7 +22,7 @@ android {
         minSdkVersion 19
         targetSdkVersion 33
         versionCode 110
```

Comment:
@shivamsharma2710 change this also to 111. Also add changelog and release notes

Reply:
@piyush-kukadiya this is done. please check.